gandalf.js

```diff
@@ -2,125 +2,127 @@
  * Extras Plugin
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.